labs/iot/gps-lab/steps/set-up-iot-hub.md

```diff
@@ -0,0 +1,39 @@
+# Set Up IoT Hub
+
+In this step you will set up IoT Hub and add a device in it.
+
+## Steps
+
+1. Go to [Azure portal]('https://portal.azure.com') and login to your subscription. 
```

Comment:
Can we refer to docs to create a hub?

Reply:
We are showing how to create IoT hub.  I think linking another document would confuse them.

However I have added a "more info" in case anyone wants to see more info.